babel.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (api) => {
+  api.cache(true);
+  api.assertVersion('^7.4.4');
+
+  const presets = [
+    '@babel/preset-env',
+  ];
+  const plugins = [
+    '@babel/plugin-transform-object-assign',
+    '@babel/plugin-proposal-object-rest-spread',
+    '@babel/plugin-transform-runtime',
+    '@babel/plugin-syntax-dynamic-import',
```

Comment:
this plugin uses dynamic imports?

Reply:
or any of these plugins for that matter?

Reply:
I removed babel altogether.